### PR TITLE
Shortened condition in Music Controller's beatmap changed function

### PR DESCRIPTION
**Pull request by marco-stephan**

### CODE REVIEW

**Comment by peppy** [comment]
99% sure this is also changing behaviour

**Review by peppy — DISMISSED** [review]

**Comment by smoogipooo** [comment]
Was the "changing of behaviour" issue previously addressed in the dev discord or something? This is still changing behaviour.

**Comment by MrTheMake** [comment]
I honestly don't see why this changes behavior. I talked to @huoyaoyuan and, if I remember correctly, he said that the assignment `direction = Transformation.None` is not needed as it is initialized to that value.

File: osu.Game/Overlays/MusicController.cs [hunk 1]
```diff
@@ -273,20 +273,21 @@ private void beatmapChanged(WorkingBeatmap beatmap)
             {
                 bool audioEquals = beatmapBacking.Value?.BeatmapInfo?.AudioEquals(current.BeatmapInfo) ?? false;
 
-                if (audioEquals)
-                    direction = TransformDirection.None;
-                else if (queuedDirection.HasValue)
+                if (!audioEquals)
```
Comment (smoogipooo) [inline osu.Game/Overlays/MusicController.cs]: Move audioEquals out of this if, and make the conditional just `if (!audioEquals)` to reduce nesting.

**Comment by smoogipooo** [comment]
Ah. I didn't notice the default value. I thought this was an instance member.

**Review by smoogipoo — CHANGES_REQUESTED** [review]

**Comment by peppy** [comment]
This is still changing behaviour, as `current` is accessed within the `if` block.

**Pull request closed without merging**